docs/io/index_unofficial.rst

```diff
@@ -0,0 +1,287 @@
+  charge per pixel, then they should be stored in the `dl1` container.
+
+See this link for more details about the `CTA High-Level Data Model
+Definitions SYS-QA/160517
+<https://jama.cta-observatory.org/perspective.req?projectId=6&docId=26528>`_
+(CTA internal).
+
+
+Containers and R1 Calibration
+=============================
+
+This is an optional section. It is included here because the R1 calibration
+is the main other inclusion camera teams may need to provide to ctapipe for
+their prototype data.
+
+If the events stored in the file are at the R0 data level, then you may need
+to create an `ctapipe.calib.camera.r1.CameraR1Calibrator` to perform the
+low-level camera-specific R1 calibration required for your events.
+
+At the R0 level the data contents are very camera specific - there may be
+additional information at this level which is used in the R1 calibration, but
+then is unused afterwards. Therefore it may be required to create a unique R0
+container for that data format. This container must contain the same
+information as `ctapipe.io.containers.R0CameraContainer`, but can contain
+additional information which may be used by the
+`ctapipe.calib.camera.r1.CameraR1Calibrator`. An example of this for
+the TargetioEventReader is the `first_cell_ids` field in
+`ctapipe.io.unofficial.targetio.containers.TargetioR0CameraContainer`.
+
+Additionally, to automate the selection of the correct
+`ctapipe.calib.camera.r1.CameraR1Calibrator` from
+`ctapipe.calib.camera.r1.CameraR1CalibratorFactory`, the
+`ctapipe.io.eventfilereader.EventFileReader.r1_calibrator` property can be
+used to specify the correct calibrator to use.
+This is then obtained from the reader, and passed to the
+`ctapipe.calib.camera.r1.CameraR1CalibratorFactory`. An example of this can be
+found in ctapipe/examples/calibration_pipeline.py. If a particular data format
+is to be used by multiple cameras, each requiring different R1 calibration,
+then this method should correctly return the right one. By default, a new
+reader will return 'NullR1Calibrator'.
+
+
+Instrument Configuration
+========================
+
+As the `ctapipe.instrument` module and "instrument database" are not fully
+developed, the camera configuration information (such as pixel positions) are
+still usually obtained from inside the hessio file.
+
+Data formats other than hessio are unlikely to contain the camera
+configuration, therefore until the `ctapipe.instrument` module and the
+"instrument database" are completed, the unofficial readers must contain other
+methods to load the instrument information into the container. This can
+involve loading data from additional files, as long as these data files are
+added to the `data_files` dictionary in setup.py.
+
+For an example of one approach to achieve this, see
+`ctapipe.io.unofficial.targetio.camera`.
+
+
+Tests and Documentation
+=======================
+
+Every unofficial reader must be thoroughly tested and documented in their
```

Comment:
No need to repeat it here, this is true for the full ctapipe project, so automatically also for every submodule.